### PR TITLE
Fix chemberta for Multitask Classification

### PR DESCRIPTION
**Pull request by riya-singh28**
## Description

Fix #(issue)

## Type of change

Please check the option that is related to your PR.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [x] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [x] Run `mypy -p deepchem` and check no errors
  - [x] Run `flake8 <modified file> --count` and check no errors
  - [x] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deepchem/models/torch_models/tests/test_chemberta.py [hunk 1]
```diff
@@ -136,3 +136,21 @@ def test_chemberta_load_weights_from_hf_hub():
     # new model's model attribute is an entirely new model initiated by AutoModel.load_from_pretrained
     # and hence it should have a different identifier
     assert old_model_id != new_model_id
+
+
+@pytest.mark.hf
+def test_chemberta_finetuning_multitask_classification():
```
Comment (ARY2260) [inline deepchem/models/torch_models/tests/test_chemberta.py:142]: please add a test for multi-task regression finetuning as well

Comment (riya-singh28) [reply deepchem/models/torch_models/tests/test_chemberta.py:142]: Added

**Comment by riya-singh28** [comment]
@ARY2260 Can you please review it again? I have added a test for multitask regression.

File: deepchem/models/torch_models/chemberta.py [hunk 2]
```diff
@@ -115,3 +121,35 @@ def __init__(self,
                                         task=task,
                                         tokenizer=tokenizer,
                                         **kwargs)
+
+    def _prepare_batch(self, batch: Tuple[Any, Any, Any]):
+        smiles_batch, y, w = batch
```
Comment (ARY2260) [inline deepchem/models/torch_models/chemberta.py:126]: add doctring and comments explaining a new prepare batch

**Review by ARY2260 — APPROVED** [review]
LGTM but needs more comments and docstrings

File: deepchem/models/torch_models/chemberta.py (hunk 2 above)
Comment (riya-singh28) [reply deepchem/models/torch_models/chemberta.py:136]: Added

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**